### ISSUE TITLE
OPML export limited due to Sailjail

### ISSUE THREAD

**Issue by RaphaelWimmer** [gpodder/gpodder-sailfish#209]
Current Sailfish OS versions enforce sailjail/firejail rules.
When one tries to export the list of podcasts into an OPML file, gpodder asks for a filename (i.e. does not present a folder selection screen).
The file is then saved into the user's home directory with a ".opml" extension.
However, the exported file never reaches the actual home directory but stays in the sailjail/firejail container and is lost when gpodder is closed.
As far as I understand the issue, gpodder would either 

- a) need a SailJail profile with an explicit permission to access the home directory or (preferrable)
- b) need to export to one of the  directories listed in `/etc/sailjail/permissions/UserDirs.permission ` (e.g., Documents, Downloads, or Music) or to the SD card 

Workaround: enter `Documents/export.opml` in the file name field when exporting the podcasts.

Alternative workaround (tried this first before looking into permissions; adjust the path to only get the .opml file you want):
```
PID=$(firejail --list | grep gpodder | cut -d: -f 1 )
firejail --cat=$PID /home/*/*.opml > /tmp/podcasts.opml
```

**Comment by Keeper-of-the-Keys** [comment]
Interesting, I guess because I have it installed over many versions I have more permissions grandfathered in, adding a Sailjail profile has been on my list of things I want to get to for a while, just didn't have priority since things seemed to be working regardless.

Can you explain a bit more why you think `b` is preferable to `a`?

**Comment by Keeper-of-the-Keys** [comment]
My view is that `a` is preferable since I also want to allow people to set their own custom download location and I am not aware of SFOS/Sailjail prompting the user for additional permissions only when needed.

**Comment by RaphaelWimmer** [comment]
> My view is that `a` is preferable since I also want to allow people to set their own custom download location and I am not aware of SFOS/Sailjail prompting the user for additional permissions only when needed.

Yes, you are right. Option b) seemed just to be the easier option.

**Comment by Keeper-of-the-Keys** [comment]
FYI - It turns out that it has to be a combination of your fix and my fix because with SailJail `${HOME}` is not accessible, I hope to merge soon and release a very long overdue release full of fixes.

**Comment by RaphaelWimmer** [comment]
Thanks for the update! I'm also following your SailJail journey [on the SFOS forums](https://forum.sailfishos.org/t/sailjail-do-i-need-bluetooth-for-media-buttons-audio-output/21218) (link to improve discoverability for others having fun with SailJail).

**Comment by Keeper-of-the-Keys** [comment]
As I'm sure you know the specific issue was resolved using the "Documents workaround" in 4.17.4

Explicit SailJail config is still WIP, I think I'm only closing this when #213 merges.
